<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LinkWindow</class>
 <widget class="QDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <layout class="QHBoxLayout">
   <item>
    <widget class="QListWidget" name="linkList"/>
   </item>
   <item>
    <layout class="QVBoxLayout">
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="linkButton">
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="unlinkButton">
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
